src/pages/GE_TokenizeCard.page

```diff
@@ -43,9 +43,17 @@
                 '}'
         })
         .then(() => {
+            isLoaded();
+
+            setTimeout(function() {
```

Comment:
Is this a potential fix or workaround for the spinner hang that we see sometimes on first load?

Reply:
Yeah, a bandaid to the issue until we can look into it more closely :\